ngraph/frontend/paddlepaddle/src/node_context.hpp

```diff
@@ -54,7 +54,7 @@ class DecoderBase {
     /// \param port_name Port name for the node
     ///
     /// \return Type of specified output port
-    virtual ngraph::element::Type get_out_port_type(const std::string& port_name) const = 0;
+    virtual ov::element::Type get_out_port_type(const std::string& port_name) const = 0;
```

Comment:
There still 'ngraph' namespace usage, is it expected?

Reply:
do you mean ngraph::HasTypeInfoMember?
yes, it's expected